packages/renderer/src/App.spec.ts

```diff
@@ -76,3 +85,28 @@ test('test /images/:id/:engineId route', async () => {
   await tick();
   expect(mocks.ImagesList).toHaveBeenCalled();
 });
+
+test('recieve context menu visible event from main', async () => {
+  render(App);
+  // send 'context-menu:visible' event
+  messages.get('context-menu:visible')?.(true);
+
+  await new Promise(resolve => setTimeout(resolve, 100));
```

Comment:
remove the await and use waitFor in the next expect condition (you want to wait until the method is called)